packages/next/src/build/webpack/plugins/wellknown-errors-plugin/parseRSC.ts

```diff
@@ -16,6 +16,8 @@ function formatRSCErrorMessage(
   const NEXT_RSC_ERR_REACT_API = /.+NEXT_RSC_ERR_REACT_API: (.*?)\n/s
   const NEXT_RSC_ERR_SERVER_IMPORT = /.+NEXT_RSC_ERR_SERVER_IMPORT: (.*?)\n/s
   const NEXT_RSC_ERR_CLIENT_IMPORT = /.+NEXT_RSC_ERR_CLIENT_IMPORT: (.*?)\n/s
+  const NEXT_RSC_ERR_CLIENT_METADATA_EXPORT =
```

Comment:
Maybe rename it to `NEXT_RSC_ERR_CLIENT_DISALLOWED_EXPORT` for more general purpose?
```suggestion
  const NEXT_RSC_ERR_CLIENT_DISALLOWED_EXPORT =
```

Reply:
🤔 I was thinking about this, but felt like it would make it cumbersome (requiring extra logic) to determine which docs to link to, when other error cases being to reuse this. Right now, the single-use nature kind of makes it easy to associate with a very specific docs page.

Just a loose opinion here. What do you think @huozhi ?

Reply:
They that sounds fair, we can refactor it later